.github/workflows/zowe-cli-plugin.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       fail-fast: false
       matrix:
         node-version: [20.x, 22.x, 24.x]
-        os: [windows-latest, ubuntu-latest, macos-13]
+        os: [windows-latest, ubuntu-latest, macos-15]

```

Comment:
Curious if we should use `macos-latest` here so we don't have to manually bump the version in the future?